tests/console/yast2_kdump.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Test scenario which configures Kdump with a YaST module
+# and checks configuration without rebooting the system.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use registration;
+use kdump_utils;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    select_console('root-console');
+    my @conf_files = @{get_test_suite_data()->{kdump}};
```

Comment:
The thing is that we are all used to the variable $test_data and as I see bellow (L:40) you are renaming it to conf_file anyway, so why not keep the tradition for better readability?

Reply:
We are using this approach in OOP with YuiRestClient as well, I'm not getting test_data key, I'm getting the array ref under kdump key, so the variable should be named properly as a list of configuration files.